.github/workflows/main.yml

```
@@ -44,6 +44,11 @@ jobs:
         run: |
           C:\msys64\usr\bin\patch.exe src\kolibri\utils\server.py patches\kolibri-utils-server.diff
 
+      # FIXME: Remove this patch when the change is released in Kolibri
```

Referencing the ticket number will help to do the work when needed:
```suggestion
      # FIXME: Remove this patch when the change is released in Kolibri
      # https://phabricator.endlessm.com/T32889
```